honeycombio/resource_trigger.go

```diff
@@ -40,7 +40,7 @@ func newTrigger() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
-			"query_json": {
+			"query": {
```

Comment:
Why change `honeycombio_trigger`.`query_json` to `query`? This fields expects a JSON representation of the query spec, this didn't change. Board also uses `query_json`, so this would be more consistent.

This is how it's usually used:
```
resource "honeycombio_trigger" "trigger" {
  # ...

  query_json = data.honeycombio_query.query.json
}
```

This is a breaking change, so if we do this it should be deliberate and consistent with the other resources.